kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableWritesSuite.scala

```diff
@@ -893,6 +978,28 @@ class DeltaTableWritesSuite extends DeltaTableWriteSuiteBase with ParquetSuiteBa
     }
   }
 
+  private def getMetadata(engine: Engine, table: Table, version: Long): Metadata = {
```

Comment:
My preference would be for this function to return an Optional[Metadata] to indicate that this can be None.